telecine/src/main/java/com/jakewharton/telecine/TelecineService.java

```diff
@@ -122,6 +123,11 @@ static Intent newIntent(Context context, int resultCode, Intent data) {
     }
   };
 
+  @Override public void onCreate() {
```

Comment:
yeah, true, but it seems weird to have one place that extends the Dagger library types to hide the code and then the rest of the places have the code themselves.